test/Microsoft.AspNet.Mvc.IntegrationTests/Models/SoftwareViewModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.AspNet.Mvc.IntegrationTests
+{
+    [ModelMetadataType(typeof(Software))]
+    public class SoftwareViewModel
```

Comment:
Currently these models are being used in two tests(TryValiateModel and ModelMetadataType) and soI did not create them as private classes.